apps/price_pusher/src/index.ts

```diff
@@ -9,6 +9,9 @@ import near from "./near/command";
 import solana from "./solana/command";
 
 yargs(hideBin(process.argv))
+  .parserConfiguration({
+    "parse-numbers": false,
+  })
```

Comment:
if you don't do this, hexadecimal strings are parsed as numbers automatically by yargs

the javascript ecosystem continues to be a dumpster fire lol